### ISSUE TITLE
Plot breaks when conventional carriers are included

### ISSUE THREAD

**Issue by Heronimonimo** [PyPSA/pypsa-eur#6]
When making a plot with conventional carriers enabled it breaks. From what I understand one of the types of line has no color defined in the tech_colors table. Unfortunately I can't find what name that line would have:
```
Traceback (most recent call last):
  File "/home/pypsa/pypsa-eur/.snakemake/scripts/tmpwavuangu.plot_network.py", line 118, in <module>
    ax=ax)
  File "/home/.conda/envs/pypsa-eur/lib/python3.6/site-packages/pypsa/plot.py", line 159, in plot
    "bus_colors must be a dictionary defining a color for each element " \
AssertionError: bus_colors must be a dictionary defining a color for each element in the second MultiIndex level of bus_sizes
[Thu Feb 14 16:32:31 2019]
Error in rule plot_network:
    jobid: 0
    output: results/plots/elec_s_10_lc1_Co2L-3H_p_nom.pdf, results/plots/elec_s_10_lc1_Co2L-3H_p_nom_ext.pdf

RuleException:
CalledProcessError in line 267 of /home/pypsa/pypsa-eur/Snakefile:
Command ' set -euo pipefail;  /home/.conda/envs/pypsa-eur/bin/python3.6 /home/pypsa/pypsa-eur/.snakemake/scripts/tmpwavuangu.plot_network.py ' returned non-zero exit status 1.
  File "/home/pypsa/pypsa-eur/Snakefile", line 267, in __rule_plot_network
  File "/home/.conda/envs/pypsa-eur/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /home/pypsa/pypsa-eur/.snakemake/log/2019-02-14T163224.663600.snakemake.log
```

**Comment by coroa** [comment]
Well, the best thing would be the change the assertion error in PyPSA to report the missing names in `bus_colors`. Unfortunately, the respective code is just now under revision (https://github.com/PyPSA/PyPSA/pull/51), so I don't want to interfere atm.

Use the following snippet (or some variant the) to find which carriers have not been defined in `tech_colors` and think up a good color for it:
```python
import pypsa
import yaml
net = pypsa.Network("results/networks/elec_s_10_lc1_Co2L-3H.nc")
with open("config.yaml") as f:
    config = yaml.load(f)
print(set(net.generators.carrier.unique()).difference(config['plotting']['tech_colors']))
```

**Comment by coroa** [comment]
Maybe fixed by #35 ?

**Comment by coroa** [comment]
Please reopen if it isn't for your case. Works here.
